src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-color: #fff;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  &__code {
+    @include text-secondary;
+  }
+
+  &__reviews {
+    @include display-position;
+    flex-direction: row;
+    margin: 15px 0 24px;
+  }
+
+  &__reviews-value {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $main-text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-title {
+    @include text-main;
+    color: $secondary-text-color;
+    font-weight: 400;
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $main-text-color;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    height: 40px;
+    border-radius: 5px;
+    border: 1px solid $accent-color;
+    color: #fff;
```

Comment:
The color `#fff` should be replaced with a variable for consistency and maintainability. Please define a variable for white color in your `variables.scss` if not already defined and use it here.